src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -206,7 +203,7 @@ export const DarCollectionTable = function DarCollectionTable(props) {
   //cancel, resubmit, and open need to be assigned as an "updateCollection" when relevant?
   //  - depends, if cancel and resubmit are locked behind modals then I only would have to pass in openCollection (only for admin and chair)
```

Comment:
Are these comments still relevant? I'm not sure I understand what they're trying to get at here.